### PR TITLE
feat(components): update styling of connect library

### PR DESCRIPTION
**Pull request by tilmx**

### CODE REVIEW

File: src/components/connect-library/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	padding: ${getSpace(SpaceSize.XL)}px ${getSpace(SpaceSize.L)}px;
+	color: ${colors.white.toString()};
+`;
+
+const StyledHeadline = styled(Headline)`
+	margin-bottom: ${getSpace(SpaceSize.M)}px;
+`;
+
+const StyledButton = styled(Button)`
+	margin: ${getSpace(SpaceSize.M)}px 0;
```
Comment (tilmx) [inline src/components/connect-library/index.tsx]: @marionebl I couldn’t get the space working here. Can you help me maybe?

Comment (tilmx) [reply src/components/connect-library/index.tsx]: Done with 86b7106

File: src/container/connect-pane-container.tsx [hunk 2]
```diff
@@ -7,28 +7,14 @@ export interface ConnectPaneProps {
...
-			>
-				Connect
-			</Components.Button>
-		</Components.Space>
-		<Components.Link color={Components.colors.grey50} onClick={props.onSecondaryButtonClick}>
-			<Components.Copy size={Components.CopySize.S}>Open Alva Example</Components.Copy>
-		</Components.Link>
-	</div>
+	<ConnectLibrary
+		headline={'Connect to a Library'}
```
Comment (marionebl) [inline src/container/connect-pane-container.tsx]: Using direct JSX notation might be a bit easier on the eyes: 

`headline="Connect to a Library"` etc

**Comment by marionebl** [comment]
Looks great. Minor nitpick I wanted you to know about, merging this anyway.

**Pull request merged**